### PR TITLE
Refactor spec file bumping to rely on rpmdev-bumpspec

### PR DESCRIPTION
**Pull request by ekohl**
This rewrites the custom logic to deal with versioning to rely on rpmdev-bumpspec. It also adds basic bumping to non-gem/npm packages.

### CODE REVIEW

File: bump_rpm.sh [hunk 1]
```diff
@@ -62,23 +75,7 @@ if [[ $CURRENT_VERSION != "$NEW_VERSION" ]] ; then
 		cd "$1"
 		GEM_NAME=$(awk '/^%global\s+gem_name/ { print $3 }' $SPEC_FILE)
 
-		spectool --list-files $SPEC_FILE | cut -d' ' -f2 | grep http | xargs --no-run-if-empty -n 1 basename | xargs --no-run-if-empty git rm
```
Comment (evgeni) [inline bump_rpm.sh:65]: won't this leave the old gem in the annex?

File: bump_rpm.sh [hunk 2]
```diff
@@ -62,23 +75,7 @@ if [[ $CURRENT_VERSION != "$NEW_VERSION" ]] ; then
...
-			echo "* Resetting release ($RELEASE) in $SPEC_FILE"
-			sed -i "s/^\(Release:\s\+\)${RELEASE}/\11/" $SPEC_FILE
-		fi
-
-		"$SCRIPT_DIR"/add_changelog.sh $SPEC_FILE <<-EOF
-		- Update to $NEW_VERSION
-		EOF
-
-		spectool --get-files $SPEC_FILE
-		spectool --list-files $SPEC_FILE | cut -d' ' -f2 | grep http | xargs --no-run-if-empty -n 1 basename | xargs --no-run-if-empty git annex add
```
Comment (evgeni) [inline bump_rpm.sh:80]: and won't this not add the new source to the annex?

File: bump_rpm.sh (hunk 1 above)
Comment (ekohl) [reply bump_rpm.sh:65]: I've moved those to bump_spec, a custom function that wraps rpmdev-bumpspec

Comment (evgeni) [reply bump_rpm.sh:65]: Ah, mhh, must have been blind (or not caffeinated).

**Review by evgeni — APPROVED** [review]

**Pull request merged**